### PR TITLE
Multiple realm files support

### PR DESCRIPTION
**Pull request by mkuchin**
Allows to import multiple realms with KeycloakContainer.withRealmImportFile() method.
Unit test included: KeycloakContainerTest.shouldImportMultipleRealms

### CODE REVIEW

**Comment by dasniko** [comment]
Thanks for your PR, I'll gonna check it...

File: src/main/java/dasniko/testcontainers/keycloak/KeycloakContainer.java [hunk 1]
```diff
@@ -13,9 +13,7 @@
 import java.io.File;
 import java.nio.charset.StandardCharsets;
 import java.time.Duration;
-import java.util.HashSet;
-import java.util.Objects;
-import java.util.Set;
+import java.util.*;
```
Comment (dasniko) [inline src/main/java/dasniko/testcontainers/keycloak/KeycloakContainer.java:16]: Please don't set `*`-imports

File: src/test/java/dasniko/testcontainers/keycloak/KeycloakContainerTest.java [hunk 2]
```diff
@@ -60,6 +60,27 @@ public void shouldImportRealm() {
         }
     }
 
+    @Test
+    public void shouldImportMultipleRealms() {
+        try (KeycloakContainer keycloak = new KeycloakContainer().
+            withRealmImportFile(TEST_REALM_JSON).
+            withRealmImportFile("another-realm.json")) {
```
Comment (dasniko) [inline src/test/java/dasniko/testcontainers/keycloak/KeycloakContainerTest.java:67]: While this is a great possibility to add multiple realm files, WDYT about an additional `.withRealmImportFiles(String... files)` method, to import multiple files with one command? I know it's "just" conveniance - but conveniance! 😉

**Review by dasniko — CHANGES_REQUESTED** [review]

File: src/main/java/dasniko/testcontainers/keycloak/KeycloakContainer.java (hunk 1 above)
Comment (mkuchin) [reply src/main/java/dasniko/testcontainers/keycloak/KeycloakContainer.java]: fixed

File: src/test/java/dasniko/testcontainers/keycloak/KeycloakContainerTest.java (hunk 2 above)
Comment (mkuchin) [reply src/test/java/dasniko/testcontainers/keycloak/KeycloakContainerTest.java]: sure, method added.

Comment (dasniko) [reply src/test/java/dasniko/testcontainers/keycloak/KeycloakContainerTest.java]: great, thanks

**Review by dasniko — APPROVED** [review]

**Comment by dasniko** [comment]
Thanks again, @mkuchin 
Your changes are contained in [Version 1.8.0](https://search.maven.org/artifact/com.github.dasniko/testcontainers-keycloak/1.8.0/jar)

**Pull request merged**